README.md

```diff
@@ -195,6 +195,10 @@ as well as the following counter metrics:
     - `delivery.batches` (one count per batch)
     - `delivery.time` (sum of delivery times in milliseconds)
     - `delivery.time2` (sum of delivery times squared)
+- latency metrics, tagged by subscriber:
+    - `latency.batches` (number of batch first delivery attempts)
+    - `latency.time.first` (sum of times from batch creation to first delivery attempt)
+    - `latency.time.last` (sum of times from batch creation to successful delivery attempt)
```

Comment:
May I suggest to use more descriptive names here? I can see why we want both metrics, but I couldn't guess by the name aloe what they were. What about `latency.time.fist_delivery_attempt` and `latency.time.successful_delivery`? A bit verbose perhaps. 🤔